sound/soc/sof/ipc4-loader.c

```diff
@@ -423,6 +423,53 @@ int sof_ipc4_reload_fw_libraries(struct snd_sof_dev *sdev)
 	return ret;
 }
 
+/**
```

Comment:
@ujfalusi the IBS*obs/1000 comment is still there in the commit message?

Reply:
Oh, I missed this, only reworded the other patch.